Lib/fontTools/svgLib/path/parser.py

```diff
@@ -209,7 +302,33 @@ def parse_path(pathdef, pen, current_pos=(0, 0)):
             last_control = control
 
         elif command == 'A':
-            raise NotImplementedError('arcs are not supported')
+            # Arc
+            radius = float(elements.pop()) + float(elements.pop()) * 1j
+            rotation = float(elements.pop())
+            arc = float(elements.pop())
+            sweep = float(elements.pop())
+            end = float(elements.pop()) + float(elements.pop()) * 1j
+
+            if not absolute:
+                if end == 0:
+                    # Guard against a situation where arc start and end being same.
+                    # That results division by zero issues in Arc parameterization.
+                    end = 0.00009
+                end += current_pos
+
+            svg_arc = Arc(current_pos, radius, rotation, arc, sweep, end)
+            arc_points = []
+            for x in range(1, 5):
+                # There are infinite points in an arc, but for our context,
+                # define the arc using 5 points(0.2, 0.4, 0.6...)
+                arc_point = svg_arc.point(x*0.2)
+                arc_points.append((arc_point.real, arc_point.imag))
+            pen.qCurveTo(
+                (current_pos.real, current_pos.imag),
+                *arc_points,
+                (end.real, end.imag)
```

Comment:
This fails on python < 3.5

`SyntaxError: only named arguments may follow *expression`

https://travis-ci.org/fonttools/fonttools/jobs/323793110#L588